Parser/pgen/__main__.py

```diff
@@ -0,0 +1,51 @@
+import os
+import sys
+import argparse
+import collections
+
+from lib2to3.pgen2 import grammar, tokenize
```

Comment:
You don't need `grammar` any more, do you?

And couldn't you use the regular stdlib's `Lib/tokenize.py`? It uses `Lib/token.py`, which is generated by `make regen-token`. That would remove the need for monkey-patching. I don't think the tokens in `tokenize.py` are actually used for anything? It's just used to parse `Grammar/Grammar`, not (in this version at least) as a source for the token values for the generated parser.